<launch>
  <!-- pcd to pointcloud "arg [filepath publish_rate]" "frame_id" "remap [input_data]" -->
  <node name="pcd_to_pointcloud" pkg="pcl_ros" type="pcd_to_pointcloud" 
        args="$(find pcl_something)/hill_map.pcd 0.1">
    <param name="frame_id" value="map"/>
    <remap from="cloud_pcd" to="/input_cloud" />
  </node>
  <!-- tf -->
  <node name="map_tf" pkg="tf" type="static_transform_publisher" args="0 0 1 0 0 3.14 map map_tf 100" />

  <node name="plane_segment"  pkg="pcl_something" type="planeDetect"/>

  <!-- rviz -->
  <node pkg="rviz" type="rviz" name="rviz"/>
</launch>

